src/care_radiology/services/dicom_service.py

```
@@ -0,0 +1,219 @@
+    def __init__(self, message):
+        super().__init__(message)
+        self.message = message
+
+
+def upload_dicom_file(patient, dcm_file):
+    if not dcm_file:
+        raise DicomUploadError("No file provided", status_code=400)
+
+    try:
+        body, content_type = encode_file_multipart_related(dcm_file)
+        upload_response = requests.post(
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)